mgc/discrim/discrimOneSample.py

```diff
@@ -0,0 +1,146 @@
+from sklearn.metrics import euclidean_distances
+from ._utils import _CheckInputs
+import numpy as np
+import random
+from .base import DiscriminabilityTest
+from scipy._lib._util import MapWrapper
+
+
+class DiscrimOneSample(DiscriminabilityTest):
+    r"""
+     A class that performs a one-sample test for whether the discriminability 
```

Comment:
better, but make the first sentence shorter and with no references. Look at the comment for `DiscrimTwoSample`

Reply:
https://github.com/neurodata/mgc/blob/2fd2f674298926ac65c4cf61f76d65b8fdd61545/mgc/discrim/discrim_one_samp.py#L8-L18